integration-tests/google-pubsub/src/test/resources/META-INF/services/org.apache.camel.quarkus.test.support.google.GoogleTestEnvCustomizer

```diff
@@ -0,0 +1 @@
+org.apache.camel.quarkus.component.google.pubsub.it.GooglePubSubCustomizer
```

Comment:
Could you please remind why we need this ?

Reply:
ok, I recall now.

Reply:
This is a registration of google-customizer through SPI, which is used to create/delete pubsub resources before/after the test. Implements `googleTestEnvCustomizer.java` from `google-tests-support` module (also part of this PR)